### PR TITLE
Resolve Wrong Label when create new "Attribute Set" (issue 24147)

### LINKED ISSUE #24147: Wrong Label when create new "Attribute Set"
### Preconditions (*)

1. Magento 2.3-develop 

### Steps to reproduce (*)

1. Go to backend
2. Store->Attribute Set
3. New Attribute Set
See the label:

image

### Expected result (*)
Should be:

image

### Actual result (*)

image

### PR DESCRIPTION
**Pull request by edenduong**
### Description (*)

1. Resolve magento/magento2#24147: Wrong Label when create new "Attribute Set" 

Solution: Check the condition, if the form doesn't have the "id", it should have label "Attribute Set Information"

### Fixed Issues (if relevant)

1. magento/magento2#24147: Wrong Label when create new "Attribute Set" 

### Manual testing scenarios (*)

1. Go to backend
2. Store->Attribute Set
3. New Attribute Set
See the label:

#### Expected result
Should be:

image
### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Catalog/Block/Adminhtml/Product/Attribute/Set/Main/Formset.php [hunk 1]
```diff
@@ -43,7 +50,11 @@ protected function _prepareForm()
 
         /** @var \Magento\Framework\Data\Form $form */
         $form = $this->_formFactory->create();
-        $fieldset = $form->addFieldset('set_name', ['legend' => __('Edit Attribute Set Name')]);
+        if (!$this->getRequest()->getParam('id', false)) {
```
Comment (Stepa4man) [inline app/code/Magento/Catalog/Block/Adminhtml/Product/Attribute/Set/Main/Formset.php]: Please consider to move this if logic to the separate method:
```
$fieldset = $form->addFieldset('set_name', ['legend' => $this->getAttributeSetLabel()]);
```
```
    private function getAttributeSetLabel()
    {
        if ($this->getRequest()->getParam('id', false)) {
            return __('Edit Attribute Set Name');
        }

        return __('Attribute Set Information');
    }
```

**Review by Stepa4man — APPROVED** [review]
Hi @edenduong,
Thanks for your contribution.

**Review by Stepa4man — CHANGES_REQUESTED** [review]
Please check my comment.

**Comment by magento-engcom-team** [comment]
Hi @Stepa4man, thank you for the review.
[ENGCOM-5644](https://jira.corp.magento.com/browse/ENGCOM-5644) has been created to process this Pull Request
:eight_spoked_asterisk: @Stepa4man, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

File: app/code/Magento/Catalog/Block/Adminhtml/Product/Attribute/Set/Main/Formset.php (hunk 1 above)
Comment (edenduong) [reply app/code/Magento/Catalog/Block/Adminhtml/Product/Attribute/Set/Main/Formset.php]: I changed the source code. Please check it for me :)  @Stepa4man

**Comment by edenduong** [comment]
I changed the source code. Please check it for me :) @Stepa4man

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-5644](https://jira.corp.magento.com/browse/ENGCOM-5644) has been created to process this Pull Request
:eight_spoked_asterisk: @dmytro-ch, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Review by Stepa4man — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @Stepa4man, thank you for the review.
[ENGCOM-5644](https://jira.corp.magento.com/browse/ENGCOM-5644) has been created to process this Pull Request

**Pull request merged**